src/index.html

```diff
@@ -7,10 +7,34 @@
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      class="form "
+      data-qa="big">
+      <input
+        type="text"
+        class="bar bar--big"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+        method="#"
+      >
+    </form>
+
+    <form
+      action="#"
+      class="form"
+      data-qa="small">
+      <input
+        type="text"
+        class="bar bar--small"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+        method="#"
+      >
+    </form>
```

Comment:
this search bar should have smaller icon, and smaller text, check the design, that's way your tests are failing